GitVersionConfig.yaml

```diff
@@ -3,3 +3,5 @@ next-version: 6.0
 branches:
   develop:
     tag: alpha
+  releases?[/-]:
```

Comment:
I have been thinking about that. I have an open issue at https://github.com/GitTools/GitVersion/issues/1018 if you want to jump in and suggest that :)